samples/charts/category-chart/annotations/_Imports.razor

```diff
@@ -1,4 +1,4 @@
-@using System.Net.Http
+﻿@using System.Net.Http
```

Comment:
you must have some VS Code extension that keeps inserting not needed whitespace in .razor files